generate-manifest.py

```diff
@@ -76,6 +77,30 @@ def edit_manifest(data, arch, branch, runtime_version):
             data['modules'].insert(0, gtk_module)
             break
 
+def sha256(filename):
+    checksum = hashlib.sha256()
+    with open(filename, 'rb') as file:
+        for data in iter(lambda: file.read(65536), b''):
+            checksum.update(data)
+    return checksum.hexdigest()
+
+def add_fonts_module(data):
+    """Add fonts module to manifest json"""
+
+    sources = [ { 'type': 'file', 'path': 'fonts-makefile', 'dest-filename': 'Makefile' } ]
```

Comment:
Since we're updating the manifest, I think we should also drop the `fonts-makefile` hack, and use `"build": "simple"`.